context/app/static/js/components/error/ErrorBody/ErrorBody.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { LightBlueLink } from 'js/shared-styles/Links';
+
+const LoginLink = () => <LightBlueLink href="/login">login</LightBlueLink>;
+const HelpEmailLink = () => (
+  <LightBlueLink href="mailto:help@hubmapconsortium.org">help@hubmapconsortium.org</LightBlueLink>
+);
+
+function ErrorBody({ errorCode, isGlobus401, isMaintenancePage }) {
+  if (isMaintenancePage) {
+    return (
+      <>
+        While the portal is under maintenance, visit the{' '}
+        <LightBlueLink href="https://hubmapconsortium.org/" target="_blank" rel="noopener noreferrer">
+          HuBMAP Consortium
+        </LightBlueLink>{' '}
+        website.
+      </>
+    );
+  }
+
+  if (errorCode === 401 && isGlobus401) {
+    return (
+      <>
+        Your credentials have expired. Please <LoginLink /> again.
+      </>
+    );
+  }
+
+  if (errorCode === 401) {
+    return (
+      <>
+        You have not been added to the HuBMAP Group on Globus. Request access at <HelpEmailLink />.
+      </>
+    );
+  }
+
+  if (errorCode === 403) {
+    return (
+      <>
+        You may not have access to this resource or you need to <LoginLink />.
```

Comment:
Can we not suggest logging in if they're already logged in?

Reply:
@tsliaw ?

Reply:
If that's not too much additional work, that should be better to not suggest logging in if they're already logged in.

Reply:
Text: